cub/test/catch2_segmented_sort_helper.cuh

```diff
@@ -125,7 +123,7 @@ public:
   void prepare_verification_data(const c2h::device_vector<key_t>& in_keys)
   {
     c2h::host_vector<key_t> h_in{in_keys};
-    keys_histogram = c2h::host_vector<std::size_t>(max_histo_size, 0);
+    keys_histogram = c2h::host_vector<std::size_t>(short_key_verification_helper<KeyT>::max_histo_size, 0);
```

Comment:
Q: Why is the extra qualification needed? `max_histo_size` is your class' member.